docs/book/src/reference/version-skew-policy.md

```diff
@@ -0,0 +1,60 @@
+# Cluster API Version and version skew support policy
+
+## Supported versions
+
+The Cluster API project maintains the Core Provider, the Kubeadm Bootstrap Provider, the Kubeadm Control Plane Provider. These run on a Management Cluster. The project aims to keep the current minor release compatible with the actively supported Kubernetes minor releases, i.e., the current release (N), N-1, and N-2. To find out the exact range of Kubernetes APIs the Management Cluster may run, see the tables below.
+
+### Core Provider - `cluster-api-controller`
+
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
+|-|-|-|-|
+|Kubernetes v1.13|✓|✓||
+|Kubernetes v1.14||✓||
+|Kubernetes v1.15||✓||
+|Kubernetes v1.16||✓|✓|
+|Kubernetes v1.17|||✓|
+|Kubernetes v1.18|||✓|
+
+The Core Provider also talk to the Kubernetes API of every Workload Cluster. The Workload Cluster's Kubernetes API version must also be compatible.
+
+In some cases, the Management Cluster is separate from the Workload Clusters. The Kubernetes API version of the Management and Workload Clusters are allowed to be different. For instance, the Management Cluster can be v1.18.2, and the Workload Clusters can be at v1.16.9 and v1.17.5.
+
+### Kubeadm Bootstrap Provider - `kubeadm-bootstrap-controller`
+
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
+|-|-|-|-|
+|Kubernetes v1.13||||
+|Kubernetes v1.14||✓||
+|Kubernetes v1.15||✓||
+|Kubernetes v1.16||✓|✓|
+|Kubernetes v1.17|||✓|
+|Kubernetes v1.18|||✓|
+
+The Kubeadm Bootstrap Provider generates configuration using the kubeadm APIs.
```

Comment:
might be you want to add that we are using v1beta1 version of the kubeadm API

Reply:
Will do, thanks!

I don't have a section on upgrading yet. Do you think there is anything CAPI users need to be aware of as v1beta1  is deprecated and eventually removed?

Reply:
There is an issue for this, but no decision yet AFAIK